src/main/java/productDependency/CreateDependentCategoryPage.java

```diff
@@ -0,0 +1,974 @@
+package productDependency;
+
+import baseClassPackage.BasePage;
+import productDependency.CreateDependentCategoryPage;
+
+import java.io.IOException;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.testng.Assert;
+import utilPackages.JavaScriptExec;
+import utilPackages.PropertyValExtractors;
+import utilPackages.WaitClass;
+import org.openqa.selenium.support.ui.Select;
```

Comment:
@mayankchhajer There is no any use of the Wait Class import

Reply:
Done